tests/test_chebyMap.py

```diff
@@ -491,6 +489,38 @@ def test_normalize(self):
             self.assertAlmostEqual(normAxis.min(), -1)
             self.assertAlmostEqual(normAxis.max(), 1)
 
+    def test_ChebyMapDM10496(self):
+        """Test for a segfault when simplifying a SeriesMap
+
+        We saw an intermittent segfault when simplifying a SeriesMap
+        consisting of the inverse of PolyMap with 2 inputs and one output
+        followed by its inverse (which should simplify to a UnitMap
+        with one input and one output). David Berry fixed this bug in AST
+        2017-05-10.
+
+        I tried this test on an older version of astshim and found that it
+        triggering a segfault nearly every time.
+        """
+        coeff_f = np.array([
+            [-1.1, 1, 2, 0],
+            [1.3, 1, 3, 1],
+        ])
+        coeff_i = np.array([
+            [1.6, 1, 3],
+            [-3.6, 2, 1],
+        ])
+        lbnd_f = [-2.0, -2.5]
+        ubnd_f = [1.5, -0.5]
+        lbnd_i = [-3.0]
+        ubnd_i = [-1.0]
+
+        for i in range(1000):
```

Comment:
I'm assuming the for loop here is to ensure that you trigger the segfault? Please document it as such, otherwise it's confusing why there's an un-used iterator in a loop here.

Also, please lift the assert into the loop (might as well check each of the returns, incase the bug manifests itself as data corruption instead of a segfault).